.env.local

```diff
@@ -14,6 +14,8 @@ NEXT_PUBLIC_BDR_MINT=BDR3oUcZLRQtufDahJskbsxwTvfWt9jiZkJPVr4kUQg2
 NEXT_PUBLIC_SOLANA_PROGRAM_ADDRESS=prg5qq3Tpr3mN8UgtVeqXYkp7QeFpHTb68ovzw2VwFp
 NEXT_PUBLIC_CERAMIC_NODE_URL=$CERAMIC_NODE_URL
 
+NEXT_PUBLIC_SENTRY_DSN=https://cb1850b4a81d4f308babf4bd9dab6742@o930852.ingest.sentry.io/6323835
```

Comment:
Why do we need it here? Since we copy this file to web app we will log errors to sentry when running app locally. Do we want that?

Reply:
But currently, we are not able to run `yarn build` because of the missing env variable. 
https://figmentio.atlassian.net/browse/LR-397?atlOrigin=eyJpIjoiZWE0OGYzZGExMjUxNDhjNGFiYmVjNWRkZjI3MjlhZGUiLCJwIjoiaiJ9

Reply:
@Necmttn Don't we need SENTRY_AUTH_TOKEN instead to run build? This is a token responsible for sending source maps to Sentry. I think you can keep that env variable but just leave it blank.

Reply:
<img width="1457" alt="image" src="https://user-images.githubusercontent.com/5212808/164428824-d8d5f617-aab0-4767-ac40-9212d1e3d458.png">
when i do that it's still failing in my local.

Reply:
I did  check my previous sentry implementations I did use sentry's GitHub integration to release on push , not on build 🧐 https://docs.sentry.io/product/releases/setup/release-automation/